src/index.ts

```diff
@@ -341,6 +317,32 @@ export default <ExportedHandler<Env>>{
 						});
 					}
 					const message = await res.json<APIMessage>();
+					const [, lastModalMessage] = await Promise.all([
+						// Create a thread from this message, asking the user for more information
+						fetch(`${RouteBases.api}${Routes.threads(message.channel_id, message.id)}`, {
+							headers: {
+								Authorization: `Bot ${env.DISCORD_BOT_TOKEN}`,
+								"Content-Type": "application/json",
+							},
+							method: "POST",
+							body: JSON.stringify({
+								name: question.substring(0, 17) + "...",
```

Comment:
An optional title field in the modal could be a nice idea for this? If not provided, we default to the start of the question name as done
(not a necessary change, could be nice though)